plugins/Satellites/src/gui/SatellitesDialog.cpp

```diff
@@ -1048,11 +1084,17 @@ void SatellitesDialog::populateInfo()
 	ui->labelRCS->setToolTip(QString("<p>%1</p>").arg(q_("Radar cross-section (RCS) is a measure of how detectable an object is with a radar. A larger RCS indicates that an object is more easily detected.")));
 	ui->labelStdMagnitude->setToolTip(QString("<p>%1</p>").arg(q_("The standard magnitude of a satellite is defined as its apparent magnitude when at half-phase and at a distance 1000 km from the observer.")));
 	// TRANSLATORS: duration
-	ui->orbitDurationSpin->setSuffix(qc_(" s","time unit"));
-	// TRANSLATORS: duration
-	ui->labelSegmentLength->setText(q_("Segment length:"));
+	QString s = qc_("s","time unit");
+	ui->orbitDurationSpin->setSuffix(QString(" %1").arg(s));
 	// TRANSLATORS: duration
-	ui->updateFrequencySpinBox->setSuffix(qc_(" h","time unit"));
+	ui->updateFrequencySpinBox->setSuffix(QString(" %1").arg(qc_("h","time unit")));
```

Comment:
Try to re-use existing qc_... entries! Better write `qc_("h", "time")` and `qc_("s", "time")`.